lib/ts/utils.ts

```diff
@@ -122,3 +124,36 @@ export function validateTheStructureOfUserInput(config: any, inputSchema: any, c
 export function frontendHasInterceptor(req: BaseRequest): boolean {
     return getRIDFromRequest(req) !== undefined;
 }
+
+/*
+ The error logger and info logger defined below use the debug lib to log error and info messages when the DEBUG env is set with the com.supertokens namespace.
+For example:
+    errLoggerWithCode[1]("Test")
+    infoLogger("some info")
+
+Output: (with DEBUG=com.supertokens:*)
+  com.supertokens:error {t: "2022-03-09T10:47:57.379Z", errCode: 1, msg: "Example error with param Test", sdkVer: "9.0.0"} +0ms
+  com.supertokens:info {t: "2022-03-09T10:47:57.380Z", msg: "Some info", sdkVer: "9.0.0"} +0ms
+
+*/
+
+// ERROR_CODE values should be the same as the key in the errorLoggerWithCode function
+const ERROR_CODE_1 = 1;
+
+let errLoggerHelper = (errorCode: number, message: string) => {
+    debug(`${SUPERTOKENS_NAMESPACE}${ERROR_ID}`)(
+        `{t: "${new Date().toISOString()}", errCode: ${errorCode}, msg: "${message}", sdkVer: "${version}"}`
+    );
+};
+
+export function infoLogger(message: string) {
+    debug(`${SUPERTOKENS_NAMESPACE}${INFO_ID}`)(`{t: "${new Date().toISOString()}", msg: "${message}"}`);
+}
+
+export let errorLoggerWithCode = {
+    1: (item: string) => {
+        errLoggerHelper(ERROR_CODE_1, "Example error with param: " + item);
```

Comment:
Don't have an example error code in the SDK. Actually use it in one place in the code